test/IDE/complete_unresolved_members.swift

```diff
@@ -28,8 +28,8 @@
 // RUN: %target-swift-ide-test -code-completion -source-filename %s -code-completion-token=UNRESOLVED_19 | %FileCheck %s -check-prefix=UNRESOLVED_1
```

Comment:
I guess this patch fixes (currently disabled) `UNRESOLVED_44`.

Reply:
Oh, good catch - it does. Let me re-enable it.